types/Webhook/WebhookCreateParameters.d.ts

```diff
@@ -1,3 +1,5 @@
 export declare interface IWebhookCreateParameters {
   url: string;
+  secret?: string;
```

Comment:
looking at the test file above, should this be `secret` or `webhook_secret`?

Reply:
Great catch! Fixed